lib/flop_phoenix/table.ex

```diff
@@ -177,13 +177,13 @@ defmodule Flop.Phoenix.Table do
         <tr
           :for={item <- @items}
           id={@row_id && @row_id.(item)}
-          {@opts[:tbody_tr_attrs]}
+          {maybe_invoke_options_callback(@opts[:tbody_tr_attrs], item)}
         >
           <%= for col <- @col do %>
             <td
               :if={show_column?(col)}
               {@opts[:tbody_td_attrs]}
-              {Map.get(col, :attrs, [])}
+              {maybe_invoke_options_callback(Map.get(col, :attrs, []), item)}
```

Comment:
@woylie I considered also adding this line down below in the `@actions` `td`s, but it wasn't strictly necessary for my use case, and I wasn't sure if it was a no-brainer to have dynamism for the `action[:attrs]`. 

If this first change is acceptable and you want me to extend it to the actions, it will be as simple as replacing what's currently line 195 (on `main`) with `{maybe_invoke_options_callback(Map.get(action, :attrs, []), item)}`

Reply:
Regarding why I had to use `col[:attrs]` here, it's because col allows the logic to be `:col`-specific.

I initially made `tbody_td_attrs` dynamic, then realized I couldn't make the attr generation column-specific at that level, as it would apply the same attrs to all `td`s in a given `td`